crates/miden-macros/tests/integration_test.rs

```diff
@@ -0,0 +1,78 @@
+#[cfg(test)]
+mod tests {
+    use miden_macros::WordWrapper;
+    use miden_objects::{Felt, FieldElement, Word};
+
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, WordWrapper)]
+    pub struct TestId(Word);
+
+    impl TestId {
+        /// Test helper to create a TestId from a Word without validation
+        pub fn new_unchecked(word: Word) -> Self {
```

Comment:
I think it'd be okay to make this an automatically derived method on `WordWrapper`.